python/pacemaker/_cts/CTS.py

```diff
@@ -178,8 +202,24 @@ def wait_for_all_nodes(self, nodes, timeout=300):
 
 
 class Process:
+    """ A class for managing a process """
+
     def __init__(self, cm, name, dc_only=False, pats=None, dc_pats=None,
                  badnews_ignore=None, triggers_reboot=False):
+        """ Create a new Process instance.
+
+            Arguments:
+
+            cm              -- A ClusterManager instance
+            name            -- The command being run
+            dc_only         -- Should this process only run on the DC?
+            pats            -- Regexes we expect to find in log files
+            dc_pats         -- Additional DC-specific regexes we expect to find
+                               in log files
+            badnews_ignore  -- Regexes for lines in the log that can be ignored
+            triggers_reboot -- Does this command cause a node to reboot?
```

Comment:
AFAICT triggers_reboot is unused except a debug log, so we could just drop it (related, fastfail is always 0, so we could drop it too)

Reply:
Removed.  I've also added a patch to get rid of fastfail too, just in case I forget when I get around to updating that file.